code/modules/research/designs/misc_designs.dm

```diff
@@ -1,543 +1,543 @@
-
```

Comment:
Unless I'm missing something, you don't need to change anything about this file and probably did it by accident. Revert the changes and this is good to be merged.